### PR TITLE
OpenShift 201 - Best Practices for Image Management

### PR DESCRIPTION
**Pull request by jayfray12**

### CODE REVIEW

File: openshift-201/image-management.md [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+--> Success
+...<em>output omitted</em>...
+</pre>
+
+As you can see there are a few resources create with the `new-app` command.  One is the `BuildConfig`.  To see the `myapp` `BuildConfig` click on the `Builds -> BuildConfigs` item in the left menu then choose `myapp` and select `YAML` or run `oc get bc/myapp -o yaml`.  You should see something similiar to the following:
+
+![buildconfig](images/image-management/buildconfig.png)
+
+1. Name of BuildConfig
+2. Defines the output.  Where the image will go after it is successfully build.
```
Comment (StevenBarre) [inline openshift-201/image-management.md:48]:
```suggestion
2. Defines the output.  Where the image will go after it is successfully built.
```
past tense

File: openshift-201/image-management.md [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+In the following exercise, you will manage application builds with OpenShift, using the source strategy with a Git input source.
+
+The following commands are used to create a new application.  The `myapp` application created is a simple Java Sprint Boot app that will display a message based on environment variables.
+
+### Create a new application 
+```bash
+oc new-app --name myapp \
+-i redhat-openjdk18-openshift:1.8 \
+ https://github.com/BCDevOps/devops-platform-workshops \ 
+ --context-dir=openshift-201/materials/image-management/sample-app
```
Comment (ShellyXueHan) [inline openshift-201/image-management.md:29]:
```suggestion
oc new-app --name myapp-shelly \
-i redhat-openjdk18-openshift:1.8 \
--context-dir=openshift-201/materials/image-management/sample-app \
https://github.com/BCDevOps/devops-platform-workshops#image-management
```

Comment (ShellyXueHan) [reply openshift-201/image-management.md:29]: better put the URL at the end to prevent issue reading context-dir option

**Comment by ShellyXueHan** [comment]
not sure if this has been discussed...
since we are asking teams to use Artifactory for app image repository, should we consider providing examples that push images to artifactory instead of local OpenShift image registry? @caggles @StevenBarre any thoughts?

File: openshift-201/image-management.md (hunk 2 above)
Comment (jayfray12) [reply openshift-201/image-management.md]: Fixed

File: openshift-201/image-management.md [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+```
+```yaml
+  tags:
+  - items:
+    - created: "2022-05-03T18:43:28Z"
+      dockerImageReference: image-registry.openshift-image-registry.svc:5000/ad204f-dev/hello-world@sha256:43378e2447d3fd0d1a8e84ac82ae88bf269d1c60ab0de29b1dc41475d5270284
+      generation: 1
+      image: sha256:43378e2447d3fd0d1a8e84ac82ae88bf269d1c60ab0de29b1dc41475d5270284
+    - created: "2022-05-03T18:10:14Z"
+      dockerImageReference: image-registry.openshift-image-registry.svc:5000/ad204f-dev/hello-world@sha256:9f388438ee6863477829e8d95cff895654030470aba5ca55a8f76a9f291c4ce2
```
Comment (mtspn) [inline openshift-201/image-management.md:386]: Make a note for participants update this command with the correct namespace

**Review by mtspn — APPROVED** [review]
Looks good, walkthrough testing complete.

File: openshift-201/image-management.md (hunk 3 above)
Comment (jayfray12) [reply openshift-201/image-management.md:386]: Done

**Comment by mtspn** [comment]
Instances of my-app could be replaced with instructions to create my-app-[username] so that lab participants are creating unique objects, and similar for the config map section

**Pull request merged**